src/main/resources/templates/index.html

```diff
@@ -2,53 +2,51 @@
     <h1>Welcome to RoadTrip Runner!</h1>
-
-    <iframe width="700" height="400" src="https://www.youtube.com/embed/JsU3RG1gYuY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-    <h3>Ready for your next adventure?  Enter your route here:</h3>
-    <form id="destination-form" method="post">
-        <label>Name Your Trip:
-            <input th:field="${trip.tripName}" th:value="${trip.tripName}">
-        </label>
-        <br>
-        <p class="error" th:errors="${trip.tripName}"></p>
-        <br>
-        <br>
-        <label th:for="startingLocation">Starting Location:
-            <input type="text" th:field="${trip.startingLocation}" th:value="${trip.startingLocation}">
-        </label>
-        <br>
-        <p class="error" th:errors="${trip.startingLocation}"></p>
-        <br>
-        <br>
-        <label th:for="endingLocation"> Destination:
-            <input type="text" th:field="${trip.endingLocation}" th:value="${trip.endingLocation}">
-        </label>
-        <br>
-        <p class="error" th:errors="${trip.endingLocation}"></p>
-        <br>
-        <br>
-        <label> Detour Radius in Miles:
-            <input type="number" th:field="${trip.detourRadius}" th:value="${trip.detourRadius}" placeholder="0">
-        </label>
-        <br>
-        <p class="error" th:errors="${trip.detourRadius}"></p>
-        <br>
-        <br>
-        <input type="submit" value="Begin!"/>
-        <br>
-        <br>
-            <p>Want to save your trips?  Click the Register button below.</p>
-        <a th:href="@{/register}" class="form-link">Register</a>
-
-    </form>
-
-
-</div>
-
+    <main class="flex-container" >
+                    <iframe class="demo-box" src="https://www.youtube.com/embed/JsU3RG1gYuY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                        <form id="destination-form" method="post">
+                            <h3>Ready for your next adventure?  Enter your route here:</h3>
+                            <br>
+                            <label>Name Your Trip:
+                                <input class="input" type="text" th:field="${trip.tripName}" th:value="${trip.tripName}">
+                            </label>
+                            <br>
+                            <p class="error" th:errors="${trip.tripName}"></p>
+                            <br>
+                            <label th:for="startingLocation">Starting Location:
+                                <input class="input directionInput" type="text" id="originInput" placeholder="starting from...">
+                            </label>
+                    <!--        th:field="${trip.startingLocation}" th:value="${trip.startingLocation}" for above-->
```

Comment:
line 21 is a necessary piece, please add back in!

Reply:
resolved